pkg/k8shandler/fluentd.go

```diff
@@ -337,6 +354,41 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateFluentdSecret() error
 	return nil
 }
 
+func newLogMetricExporterContainer(cluster *logging.ClusterLogging) v1.Container {
+	collectionSpec := logging.CollectionSpec{}
+	if cluster.Spec.Collection != nil {
+		collectionSpec = *cluster.Spec.Collection
+	}
+	resources := collectionSpec.Logs.LogMetricExporterSpec.Resources
+	if resources == nil {
```

Comment:
remove me.  we will not default resources for the container if they are not defined